evla_mcast/scan_config.py

```diff
@@ -54,8 +61,11 @@ def has_vci(self):
     def has_obs(self):
         return self.obs is not None
 
+    def has_ant(self):
+        return self.obs is not None
+
     def is_complete(self):
-        return self.has_vci() and self.has_obs()
+        return self.has_vci() and self.has_obs() and self.has_ant
```

Comment:
I just sent you email about this, but self.has_ant here needs to be self.has_ant().  Or we could change all the has_* functions to properties, this might be more consistent with the use of properties elsewhere in the class?  Either way is OK with me.